src/Button/Button.tsx

```diff
@@ -31,6 +31,11 @@ const Button = styled(ButtonBase)<ButtonBaseProps & SxProp>`
     background-color: ${get('colors.btn.bg')};
     border-color: ${get('colors.btn.border')};
   }
+  &.selected,
+  &[aria-selected='true'] {
```

Comment:
👍 good call on the `=true`! I've done the mistake of just putting `aria-selected` in the selector and catching `=false coz react 😅